src/ios/SitumCordovaPlugin.xcodeproj/project.pbxproj

```diff
@@ -277,6 +284,78 @@
 			path = Frameworks;
 			sourceTree = "<group>";
 		};
+		0D5CFD39212C275E00C6E436 /* CDVPlugins */ = {
+			isa = PBXGroup;
+			children = (
+				0D5CFD3A212C275E00C6E436 /* CDVGestureHandler */,
+				0D5CFD3D212C275E00C6E436 /* CDVIntentAndNavigationFilter */,
+				0D5CFD40212C275E00C6E436 /* CDVHandleOpenURL */,
+				0D5CFD43212C275E00C6E436 /* CDVLogger */,
+				0D5CFD46212C275E00C6E436 /* CDVLocalStorage */,
+				0D5CFD49212C275E00C6E436 /* CDVUIWebViewEngine */,
+			);
+			name = CDVPlugins;
+			path = "../../../../../situm-cordova-plugin2/src/ios/Classes/Private/CDVPlugins";
```

Comment:
Path should not reference to `situm-cordova-plugin2` as this will break the compilation if the project folder is not named exactly like that